core-services/prow/02_config/generate-boskos.py

```diff
@@ -47,7 +47,13 @@
         'us-east-1': 8,
         'us-east-2': 8,
         'us-west-2': 8
-    }
+    },
+    'aws-2-quota-slice': {
+        'us-east-1': 35,
+        'us-east-2': 35,
+        'us-west-1': 35,
+        'us-west-2': 35,
```

Comment:
How did these numbers came to be? Is this a guess? We should be able to compute these numbers more precisely.

Reply:
Thats a guess. DPP team can only compute those numbers but there is nothing running there now. So we can start with these numbers and check with the rehearsals. Shall I lower them?

Reply:
Fine to lower to 10 or something and unblock yourself on rehearsals, to check that the release migration work is correct, while you calculate these more correctly.

Reply:
updated